Realm/Realm/Realm.csproj

```diff
@@ -33,19 +33,28 @@
   <ItemGroup>
     <EmbeddedResource Include="Properties\Realm.rd.xml" />
     <AdditionalFiles Include="$(ProjectDir)..\..\stylecop.json" />
-    <Content Include="RealmWrappersReferences.props" Pack="true" PackagePath="build\Realm.props" />
     <None Update="Helpers\Operator.tt">
       <Generator>TextTemplatingFileGenerator</Generator>
       <LastGenOutput>Operator.cs</LastGenOutput>
     </None>
     <!-- iOS -->
-    <None Update="ios-dllmap.config" Pack="true" PackagePath="native\ios\$(AssemblyName).dll.config" />
-    <None Include="..\..\wrappers\build\iOS\$(Configuration)\realm-wrappers.xcframework\**">
+    <Content Include="ios-dllmap.config" Pack="true">
+      <PackagePath>contentFiles\any\xamarinios\$(AssemblyName).dll.config</PackagePath>
+      <PackageCopyToOutput>true</PackageCopyToOutput>
+    </Content>
+    <Content Include="wrappers-props\xamarinios\wrappers.props" Pack="true" PackagePath="build\xamarinios\Realm.props" />
```

Comment:
Is `build/xamarinios` the folder being used both for Xamarin.iOS and net6-ios TFMs?

Reply:
No, it's only the Xamarin.iOS TFM that uses it.